ratis-examples/src/main/java/org/apache/ratis/examples/counter/server/CounterServer.java

```diff
@@ -78,24 +78,41 @@ public void close() throws IOException {
     server.close();
   }
 
-  public static void main(String[] args) throws IOException {
-    if (args.length < 1) {
-      System.err.println("Usage: java -cp *.jar org.apache.ratis.examples.counter.server.CounterServer {serverIndex}");
-      System.err.println("{serverIndex} could be 1, 2 or 3");
+  public static void main(String[] args) {
+    try {
+      //get peerIndex from the arguments
+      if (args.length == 1) {
```

Comment:
This judgment condition seems to be wrong，args.length != 1

Reply:
Good catch!  Will update it.